stories/module-analytics-setup-with-ga4.stories.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Analytics Setup stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { storiesOf } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import ModuleSetup from '../assets/js/components/setup/ModuleSetup';
+import * as fixtures from '../assets/js/modules/analytics/datastore/__fixtures__';
+import * as ga4Fixtures from '../assets/js/modules/analytics-4/datastore/__fixtures__';
+import { MODULES_ANALYTICS, ACCOUNT_CREATE, PROVISIONING_SCOPE } from '../assets/js/modules/analytics/datastore/constants';
+import { CORE_SITE } from '../assets/js/googlesitekit/datastore/site/constants';
+import { CORE_USER } from '../assets/js/googlesitekit/datastore/user/constants';
+import { MODULES_ANALYTICS_4 } from '../assets/js/modules/analytics-4/datastore/constants';
+import {
+	WithTestRegistry,
+	createTestRegistry,
+	provideModules,
+	provideModuleRegistrations,
+} from '../tests/js/utils';
+
+function Setup( props ) {
+	return (
+		<WithTestRegistry
+			features={ [ 'ga4setup' ] }
```

Comment:
Unlike in my other PR https://github.com/google/site-kit-wp/pull/3383/files#diff-04df598d47fecc4c707dcb45aa88f142b5b322333151939b084883795c6ddaccR132

This works for setting the feature flag :thinking:

Reply:
It works because you rely on `useFeature` hook instead of on `isFeatureEnabled`. For some reason, they don't work well together, but it's a matter of a separate issue.